ext/standard/basic_functions.stub.php

```diff
@@ -2136,9 +2136,9 @@ function crypt(#[\SensitiveParameter] string $string, string $salt): string {}
 #ifdef HAVE_STRPTIME
 /**
  * @return array<string, int|string>|false
- * @deprecated
  * @refcount 1
  */
+#[\Deprecated]
```

Comment:
```suggestion
#[\Deprecated(since: '8.2', message: 'use date_parse_from_format() (for locale-independent parsing), or IntlDateFormatter::parse() (for locale-dependent parsing) instead')]
```